packages/paste-website/src/components/tokens-list/TokensExample.tsx

```diff
@@ -50,7 +71,13 @@ interface TextColorBoxProps extends TextColorBoxProp {
 export const TextColorBox: React.FC<TextColorBoxProps> = ({boxColor, color}) => {
   const theme = useTheme();
   const colorFn = Color(boxColor);
-  const isInverse = colorFn.isLight();
+  let isInverse;
+  if (color !== 'colorTextWeaker') {
+    isInverse = colorFn.isLight();
+  }
+  if (color === 'colorTextInverseWeaker') {
+    isInverse = false;
+  }
```

Comment:
The accessibiilty ratings due to this change seems wrong to me 
![image](https://user-images.githubusercontent.com/368249/89238397-c38a5180-d5aa-11ea-83a2-5e2931e229a6.png)

Reply:
For clarity white on almost black should definitely be AAA not a fail.

Reply:
You're right. That should be AAA. When I revert the change, I'm still getting the same results.

![image](https://user-images.githubusercontent.com/1350081/89248063-fc86ee00-d5cb-11ea-9c38-bdfe1de3a657.png)

Reply:
I figured it out. Should fixed now.